test/core/Do.Test.Architecture/Architecture/Application/RunningAnApplication.cs

```diff
@@ -10,8 +10,8 @@ public void Application_collects_phases_from_all_layers()
         var phase1 = MockMe.APhase();
         var phase2 = MockMe.APhase();
         var phase3 = MockMe.APhase();
-        var layer1 = MockMe.ALayer(phases: new[] { phase1, phase2 });
-        var layer2 = MockMe.ALayer(phase: phase3);
+        var layer1 = MockMe.ALayer(id: "Do.Test.Architecture.Application.LayerA", phases: new[] { phase1, phase2 });
+        var layer2 = MockMe.ALayer(id: "Do.Test.Architecture.Application.LayerB", phase: phase3);
```

Comment:
use guid when mocking a layer to avoid setting a unique id where it is irrelevant